client/my-sites/upgrades/checkout-thank-you/index.jsx

```diff
@@ -49,7 +48,7 @@ function getPurchases( props ) {
 	return props.receipt.data.purchases;
 }
 
-var CheckoutThankYou = React.createClass( {
+const CheckoutThankYou = React.createClass( {
```

Comment:
Maybe we can consider moving to ES6 gradually in this case since we will have so many PR touching the code at the same time? Or is that a bad idea?